.github/workflows/ci-wheels.yml

```diff
@@ -117,10 +117,10 @@ jobs:
     name: "Show artifacts"
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/download-artifact@v3
+    - uses: actions/download-artifact@v4
       with:
-        name: pypi-artifacts
         path: ${{ github.workspace }}/dist
+        merge-multiple: true
```

Comment:
Not providing a `name` prompts ALL artifacts to be downloaded. Using `merge-multiple` 'unpacks' the contents of all artifacts into the directory. This keeps behaviour as it used to be; compare:

- [A run from the current `v3` implementation](https://github.com/SciTools/cf-units/actions/runs/11074565922/job/30773802900?pr=491#step:3:1)
- [The latest run on this PR](https://github.com/SciTools/cf-units/actions/runs/11075703359/job/30777346128?pr=493#step:3:1)